### ISSUE TITLE
no subtitles on Kodi 20.0

### ISSUE THREAD

**Issue by david63236** [add-ons/plugin.video.vtm.go#346]
The subtitles do not appear (after the automatic update to Kodi 20.0 on Android TV)

**Comment by mediaminister** [comment]
I found a bug.
Please test
Kodi 19/20: [plugin.video.vtm.go-1.3.3+matrix.1.zip](https://github.com/add-ons/plugin.video.vtm.go/files/10479905/plugin.video.vtm.go-1.3.3%2Bmatrix.1.zip)
Kodi 18: [plugin.video.vtm.go-1.3.3.zip](https://github.com/add-ons/plugin.video.vtm.go/files/10479904/plugin.video.vtm.go-1.3.3.zip)

**Comment by david63236** [comment]
Will test that... 
Please note that i didn't have that problem on Kodi 19.3 (before automatic update), the subtitles appeared.
I must say that after a chapter change, wrong subtitles (I thing from other chapter) appeared.
-- may take a while, I remember that the automatic update from 18 to 19 had problems to,
a fresh installation was needed to solve these problems, so I'm now reinstalling Kodi 20.

The subtitles appeared, but after a chapter change
Maybe a fresh install (was needed when Kodi 18 was updated to Kodi 19)

**Comment by david63236** [comment]
Sorry for the delay.
I have to many problems (all video add-ons) with Kodi 20.0 on Android TV, I think my TV has to little RAM or processor power.
So I reinstalled Kodi 19.3 and all is working.

After that, I did a fresh install of Kodi 20.0 on Windows and installed the VTMgo add-on from repository,
this gives the same problem: The subtitles do not appear

After that, I installed your test version and yes, the subtitles appear, but....
when moving forward or backward in the same chapter subtitles are OK,
when moving to an other chapter (using the >| button) subtitles are WRONG,
rewinding to the begin of the video and subtitles are OK,
moving to end of first chapter and let it play starts next chapter and subtitles are OK.
so, the problem only occurs when changing chapter with the >| and |< buttons.
